### PR TITLE
Add blog post about upcoming native packaging changes.

### PR DESCRIPTION
**Pull request by Ferroin**
Tentatively planned for posting next Wednesday.

### CODE REVIEW

**Comment by andrewm4894** [comment]
Sweet! @Ferroin welcome to the blog, lets go!!!

File: blog/2023-03-08-split-plugin-packages.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+- The eBPF plugin (in the new `netdata-plugin-ebpf` and `netdata-ebpf-legacy-code` packages)
+- The apps plugin (in the new `netdata-plugin-apps` package)
+
+## Why are we making this change?
+
+With our current approach to packaging Netdata, we actually ship a _lot_ of code that most users never actually use.
+
+For our users, this means that:
+
+- Netdata takes up more disk space than it really needs to.
```
Comment (andrewm4894) [inline blog/2023-03-08-split-plugin-packages.md:40]: Any easy way to quantify the impact here on some sort of "typical" installation? Just curious if is like a major major change or more like incremental best practice type improvements so maybe not worth lots of effort to try get some representative number.

File: blog/2023-03-08-split-plugin-packages.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+In this case, you don’t need to do anything, and should not be affected by this change.
+
+### If you are using our official DEB packages
+
+#### New installs
+
+Most new installs should be unaffected by this change. There are two exceptions:
+
+1.  If you need the nfacct, perf, or slabinfo plugins, or one of the collectors provided by the `charts.d` plugin,
+    you will need to manually install the associated packages.
```
Comment (andrewm4894) [inline blog/2023-03-08-split-plugin-packages.md:65]: wondering if we have learn docs or something we could link to here?

File: blog/2023-03-08-split-plugin-packages.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+2.  If you have your system configured to not install recommended packages by default, then you will need to either
+    manually install all the external plugin packages you require (or, alternatively, check the second option below
+    for existing installs to pull in the default set).
+
+#### Existing installs
+
+Updating from a version from before this change will result in the external plugins being essentially removed
+from your system and needing to be manually installed if required.
+
+This is a result of a limitation in how APT handles soft dependencies on updates (in particular, it simply doesn’t).
```
Comment (andrewm4894) [inline blog/2023-03-08-split-plugin-packages.md:75]: lol :)

**Comment by andrewm4894** [comment]
@Ferroin should we add a final section on how/where to get help if needed - e.g open a GH discussion or community thread maybe?

File: blog/2023-03-08-split-plugin-packages.md (hunk 2 above)
Comment (Ferroin) [reply blog/2023-03-08-split-plugin-packages.md:65]: Not currently, but the documentation pages for all the affected collectors will be updated as part of the merge of the associated PR to include information on what package needs to be installed.

File: blog/2023-03-08-split-plugin-packages.md (hunk 1 above)
Comment (Ferroin) [reply blog/2023-03-08-split-plugin-packages.md:40]: In general, it’s not likely to be more than a few kB to a few dozen MB.

File: blog/2023-03-08-split-plugin-packages.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  not needed.
+- Their system presents a larger attack surface than it would otherwise.
+
+By splitting our external data collection plugins out to individual packages like this, we’re making it easier
+for users to ensure that they only have what they actually need on their system, helping to resolve all of the
+above-mentioned issues.
+
+Additionally, this helps us reduce the overhead involved in hosting our official packages.
+
+## What do I need to do?
```
[the remark this answers is not in the document]
Comment (Ferroin) [reply blog/2023-03-08-split-plugin-packages.md:52]: Good point. I’ll add some info here on how to figure this out.

**Comment by Ferroin** [comment]
@lupimiguel I would appreciate if I could get some feedback from you on this, especially on the point @andrewm4894 mentioned above about possibly adding help links.

**Review by lupimiguel — APPROVED** [review]

**Review by andrewm4894 — APPROVED** [review]

**Pull request merged**